src/Orc.Controls/Controls/LogViewer/Views/LogViewerControl.xaml

```diff
@@ -9,7 +9,9 @@
     <Grid>
         <RichTextBox x:Name="LogRecordsRichTextBox" 
                      IsReadOnly="True" 
-                     IsDocumentEnabled="True" 
+                     IsDocumentEnabled="True"
+                     BorderThickness="0"
+                     Margin="0"
```

Comment:
Not sure if we should change the margin as well. This will change the size of the control to not be the same as others?

Reply:
I think it's an child control... we should count only Control Level margins
![image](https://user-images.githubusercontent.com/5163834/125115235-b8fa0580-e0f3-11eb-86f8-c7b882ebff6e.png)

Reply:
i think it's ok as long as LogViewer control itself a fits into composition

Reply:
@ceo1647 , please check the example screenshot in the description, it looks inconsistent.

Reply:
yes...i was talking about it... Dmitry fixed it
![image](https://user-images.githubusercontent.com/5163834/125290720-4c684c00-e329-11eb-8d33-f25258f66d4b.png)